db/seeds.rb

```diff
@@ -5,10 +5,32 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-Wizard.find_or_create_by!(name: "Gandalf the White", description: "Hes got to be top ten maybe 5")
-Wizard.find_or_create_by!(name: "Gandalf the Grey", description: "Hes great and all but he hidding")
-Wizard.find_or_create_by!(name: "Gwydion", description: "Hes norse")
-Wizard.find_or_create_by!(name: "Merlin", description: "Super famouse wizard")
-Wizard.find_or_create_by!(name: "Jafar", description: "Hes from aladdin")
-Wizard.find_or_create_by!(name: "Maugris", description: "Hes enchants matter in france")
-Wizard.find_or_create_by!(name: "Saruman", description: "Hes fights Gandalf")
+require 'factory_bot'
+
+FactoryBot.define do
+  factory :user do
+    sequence(:user_name) { |n| "user#{n}"}
+    sequence(:email) {|n| "user#{n}@example.com" }
+    password 'password'
+    password_confirmation 'password'
+    role 'member'
+  end
+end
+
+gandalfa = Wizard.find_or_create_by!(name: "Gandalf the White", description: "Hes got to be top ten maybe 5")
```

Comment:
nicely done here. I love that you used `find_of_create_by!`

I would consider `gandalf_a` as a variable name to adhere to ruby conventions.